### PR TITLE
9.0 mig purchase fiscal position update

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

File: purchase_fiscal_position_update/models/purchase.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+from openerp import models, api, _
+
+
+class purchase_order(models.Model):
+    _inherit = "purchase.order"
+
+    @api.onchange('fiscal_position_id')
+    def onchange_fiscal_position_id(self):
+        for line in self.order_line:
```
Comment (pedrobaeza) [inline purchase_fiscal_position_update/models/purchase.py:18]: Do:

```
for line in self.order_line:
    temp_line = line.new(line._convert_to_write(line._cache))
    temp_line.onchange_product_id()
    line.taxes_id = temp_line.taxes_id
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Change in the approach (only drafted - to be tested)

**Comment by pedrobaeza** [comment]
Check also Travis status

File: purchase_fiscal_position_update/models/purchase.py (hunk 1 above)
Comment (angelmoya) [reply purchase_fiscal_position_update/models/purchase.py:18]: This way is not working, and using cache is not direct, so it's more clear do like @alexis-via on #280

File: purchase_fiscal_position_update/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from openerp import models, api, _
+
+
+class purchase_order(models.Model):
+    _inherit = "purchase.order"
+
+    @api.onchange('fiscal_position_id')
+    def fiscal_position_change(self):
+        '''Function executed by the on_change on the fiscal_position_id field
```
Comment (pedrobaeza) [inline purchase_fiscal_position_update/models/purchase.py]: Double quote instead of simple for docstring according PEP257

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Include tests

File: purchase_fiscal_position_update/models/purchase.py (hunk 2 above)
Comment (angelmoya) [reply purchase_fiscal_position_update/models/purchase.py]: Fixed

**Comment by angelmoya** [comment]
Tests added

**Comment by pedrobaeza** [comment]
Travis is still red

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As this one is the same as 10.0 version and it has another author, I merge both.

**Pull request merged**